src/react/Stories.js

```diff
@@ -26,19 +26,12 @@ export default class Stories extends Component {
     }
 
     return (
-      <div
-        {...rest}
-      >
-        {names.map((name) => (
-          <Fragment
-            key={name}
-          >
+      <div>
+        {names.map(name => (
+          <Fragment key={name}>
             <h2>{name}</h2>
 
-            <Story
-              kind={kind}
-              name={name}
-            />
+            <Story kind={kind} name={name} {...rest} />
```

Comment:
i think we want to pass `...rest` onto the top-level div here.

in general, when I pass something like `className` or `style` to a component, I expect them to be assigned to that component's container element.

Reply:
one option would be to take in an additional optional prop `storyProps` that is an object that will get spread for each story.

Reply:
✅